docs/user-interface/graphics/colors.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Colors"
+description: "The .NET MAUI Color class, in the Microsoft.Maui.Graphics namespace, lets you specify colors as RGB values, HSL values, HSV values, or with a color name."
+ms.date: 08/31/2021
+---
+
+# Colors
+
+<!-- Sample link goes here -->
+
+The `Color` class, in the `Microsoft.Maui.Graphics` namespace, lets you specify colors as Red-Green-Blue (RGB) values, Hue-Saturation-Luminosity (HSL) values, Hue-Saturation-Value (HSV) values, or with a color name. An Alpha channel is also available to indicate transparency.
+
+`Color` objects can be created with `Color` constructors, which can be used to specify a gray shade, an RGB value, or an RGB value with transparency. In all cases, arguments are `float` values ranging from 0 to 1.
+
+> [!NOTE]
+> The default `Color` constructor creates a black `Color` object.
```

Comment:
Also, Color.Default (from Xamarin.Forms) not exists here. The comparison to detect if a color property has not been initialized is:

`myColor == null`

Instead:

`myColor == Color.Default`

I am thinking in devs coming from Xamarin.Forms, this kind of details could help in migrations, etc.

Reply:
I'll save this for the migration doc.